utils/releaser/src/ReleaseWorker/Release/CreateAndPushGitTagsExceptProjectBaseReleaseWorker.php

```diff
@@ -94,6 +94,12 @@ public function work(Version $version): void
             }
         }
 
+        $this->symfonyStyle->note(sprintf(
+            'When you do not have saved github credentials you may want to cache them temporarily to prevent asking them for each repository.'
```

Comment:
```suggestion
            'In case you do not have saved GIT credentials you may want to cache them temporarily so you do not need to fill them for each repository.'
```